js/app/ui/module/system_killboard.js

```diff
@@ -727,7 +727,7 @@ define([
          */
         static initWebSocket(){
             if(!SystemKillboardModule.ws){
-                SystemKillboardModule.ws = new WebSocket('wss://zkillboard.com:2096');
+                SystemKillboardModule.ws = new WebSocket('wss://zkillboard.com/websocket');
```

Comment:
Not sure if you had this working on your instance but the correct URL seems to be `wss://killboard.com/websocket/` otherwise you get a permanent redirect and the UI shows as _Offline_ (Red dot).

Reply:
Hey, yes you are correct, it requires the trailing slash, works fine in our instance as `wss://killboard.com/websocket/`

Reply:
Don't you need the "z" in zkillboard? I am still getting my apache server running, so I shamefully, did not get to test.

But when I test a websocket connection on the command line with NodeJS, I need to use: wss://**z**killboard.com/websocket/

Reply:
I have added the trailing slash though. Thank you guys for catching!

Reply:
Yep, i just copy/pasted from gparent's comment. Full line in my working fork is: 

`SystemKillboardModule.ws = new WebSocket('wss://zkillboard.com/websocket/');`

Reply:
I edited my comment. On my fork I have the working link and rebuilt the static files.

Reply:
All good. Thanks guys!